src/private/interfaces.ts

```diff
@@ -110,6 +121,11 @@ export interface FilePreviewParameters {
    * Optional; indicates the mode in which file should be opened. Takes precedence over edit mode.
    */
   viewerAction?: ViewerActionTypes;
+
+  /**
+   * Optional; indicates where the file should be viewed when opened.
+   */
+  fileViewMode?: FileViewModes;
```

Comment:
Why are we giving this control to the app? Should Teams be the one that decides how to open the file based on the user's preference?

Reply:
So currently, a user can explicitly choose where to view a particular file while opening it. By adding this field apps can provide this ability.
In case this is not provided, teams will decide based on the set the preference. @ydogandjiev

Reply:
Actually undoing this. For now the underlying app can provide this functionality. We can support this when we see more use cases.
For now teams will decide based on user preference. Will be undoing the changes.